2024-09-02 16:02:15,136 INFO    StreamThr :21560 [internal.py:wandb_internal():85] W&B internal server running at pid: 21560, started at: 2024-09-02 16:02:15.135864
2024-09-02 16:02:15,138 DEBUG   HandlerThread:21560 [handler.py:handle_request():158] handle_request: status
2024-09-02 16:02:15,146 INFO    WriterThread:21560 [datastore.py:open_for_write():87] open: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\run-owb2fhm6.wandb
2024-09-02 16:02:15,147 DEBUG   SenderThread:21560 [sender.py:send():391] send: header
2024-09-02 16:02:15,198 DEBUG   SenderThread:21560 [sender.py:send():391] send: run
2024-09-02 16:02:15,755 INFO    SenderThread:21560 [dir_watcher.py:__init__():211] watching files in: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\files
2024-09-02 16:02:15,755 INFO    SenderThread:21560 [sender.py:_start_run_threads():1200] run started: owb2fhm6 with start time 1725260535.13786
2024-09-02 16:02:15,790 DEBUG   HandlerThread:21560 [handler.py:handle_request():158] handle_request: run_start
2024-09-02 16:02:15,818 DEBUG   HandlerThread:21560 [system_info.py:__init__():26] System info init
2024-09-02 16:02:15,818 DEBUG   HandlerThread:21560 [system_info.py:__init__():41] System info init done
2024-09-02 16:02:15,818 INFO    HandlerThread:21560 [system_monitor.py:start():194] Starting system monitor
2024-09-02 16:02:15,818 INFO    SystemMonitor:21560 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-09-02 16:02:15,818 INFO    HandlerThread:21560 [system_monitor.py:probe():214] Collecting system info
2024-09-02 16:02:15,821 INFO    SystemMonitor:21560 [interfaces.py:start():188] Started cpu monitoring
2024-09-02 16:02:15,822 INFO    SystemMonitor:21560 [interfaces.py:start():188] Started disk monitoring
2024-09-02 16:02:15,823 INFO    SystemMonitor:21560 [interfaces.py:start():188] Started gpu monitoring
2024-09-02 16:02:15,829 INFO    SystemMonitor:21560 [interfaces.py:start():188] Started memory monitoring
2024-09-02 16:02:15,844 INFO    SystemMonitor:21560 [interfaces.py:start():188] Started network monitoring
2024-09-02 16:02:15,847 DEBUG   HandlerThread:21560 [system_info.py:probe():152] Probing system
2024-09-02 16:02:15,850 DEBUG   HandlerThread:21560 [system_info.py:_probe_git():137] Probing git
2024-09-02 16:02:15,850 ERROR   gpu       :21560 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-09-02 16:02:15,851 ERROR   gpu       :21560 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-09-02 16:02:15,938 DEBUG   HandlerThread:21560 [system_info.py:_probe_git():145] Probing git done
2024-09-02 16:02:15,938 DEBUG   HandlerThread:21560 [system_info.py:probe():200] Probing system done
2024-09-02 16:02:15,938 DEBUG   HandlerThread:21560 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.19045-SP0', 'python': '3.9.19', 'heartbeatAt': '2024-09-02T07:02:15.847667', 'startedAt': '2024-09-02T07:02:15.124895', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'C:\\Users\\LimJaesung\\Documents\\Git\\VAE\\convVAE\\main.py', 'codePathLocal': 'main.py', 'codePath': 'convVAE\\main.py', 'git': {'remote': 'https://github.com/ImJaeSung/VAE.git', 'commit': 'f1aa7205610e135d68d576a1026ad8fb607f883d'}, 'email': None, 'root': 'C:/Users/LimJaesung/Documents/Git/VAE', 'host': 'DESKTOP-5J5VKKC', 'username': 'LimJaesung', 'executable': 'C:\\Users\\LimJaesung\\anaconda3\\envs\\impute\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 6, 'cpu_freq': {'current': 3000.0, 'min': 0.0, 'max': 3000.0}, 'cpu_freq_per_core': [{'current': 3000.0, 'min': 0.0, 'max': 3000.0}], 'disk': {'/': {'total': 237.79767227172852, 'used': 164.9908676147461}}, 'gpu': 'NVIDIA GeForce GTX 1050', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1050', 'memory_total': 2147483648}], 'memory': {'total': 15.915508270263672}}
2024-09-02 16:02:15,938 INFO    HandlerThread:21560 [system_monitor.py:probe():224] Finished collecting system info
2024-09-02 16:02:15,938 INFO    HandlerThread:21560 [system_monitor.py:probe():227] Publishing system info
2024-09-02 16:02:15,938 DEBUG   HandlerThread:21560 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-09-02 16:02:15,955 ERROR   HandlerThread:21560 [system_info.py:_save_conda():223] Error saving conda packages: [WinError 2] 지정된 파일을 찾을 수 없습니다
Traceback (most recent call last):
  File "C:\Users\LimJaesung\anaconda3\envs\impute\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 216, in _save_conda
    subprocess.call(
  File "C:\Users\LimJaesung\anaconda3\envs\impute\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\LimJaesung\anaconda3\envs\impute\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\LimJaesung\anaconda3\envs\impute\lib\subprocess.py", line 1436, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] 지정된 파일을 찾을 수 없습니다
2024-09-02 16:02:15,957 DEBUG   HandlerThread:21560 [system_info.py:_save_conda():224] Saving conda packages done
2024-09-02 16:02:15,957 INFO    HandlerThread:21560 [system_monitor.py:probe():229] Finished publishing system info
2024-09-02 16:02:15,957 DEBUG   SenderThread:21560 [sender.py:send():391] send: files
2024-09-02 16:02:15,957 INFO    SenderThread:21560 [sender.py:_save_file():1466] saving file wandb-metadata.json with policy now
2024-09-02 16:02:16,099 DEBUG   HandlerThread:21560 [handler.py:handle_request():158] handle_request: python_packages
2024-09-02 16:02:16,099 DEBUG   SenderThread:21560 [sender.py:send_request():418] send_request: python_packages
2024-09-02 16:02:16,099 DEBUG   HandlerThread:21560 [handler.py:handle_request():158] handle_request: stop_status
2024-09-02 16:02:16,100 DEBUG   SenderThread:21560 [sender.py:send_request():418] send_request: stop_status
2024-09-02 16:02:16,101 DEBUG   HandlerThread:21560 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-02 16:02:16,404 DEBUG   SenderThread:21560 [sender.py:send():391] send: telemetry
2024-09-02 16:02:16,405 DEBUG   SenderThread:21560 [sender.py:send():391] send: config
2024-09-02 16:02:16,406 DEBUG   SenderThread:21560 [sender.py:send():391] send: telemetry
2024-09-02 16:02:16,765 INFO    Thread-16 :21560 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\files\output.log
2024-09-02 16:02:16,765 INFO    Thread-16 :21560 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\files\requirements.txt
2024-09-02 16:02:16,765 INFO    Thread-16 :21560 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\files\wandb-metadata.json
2024-09-02 16:02:16,765 INFO    Thread-16 :21560 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\LimJaesung\Documents\Git\VAE\convVAE\wandb\run-20240902_160215-owb2fhm6\files\conda-environment.yaml
2024-09-02 16:02:16,814 INFO    wandb-upload_0:21560 [upload_job.py:push():130] Uploaded file C:\Users\LIMJAE~1\AppData\Local\Temp\tmprfft7bmtwandb\78bqfzo8-wandb-metadata.json
2024-09-02 16:02:17,868 ERROR   gpu       :21560 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-09-02 16:02:17,868 ERROR   gpu       :21560 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
